packages/docusaurus/src/client/App.tsx

```diff
@@ -16,14 +16,19 @@ import BaseUrlIssueBanner from './BaseUrlIssueBanner';
 import SiteMetadataDefaults from './SiteMetadataDefaults';
 import Root from '@theme/Root';
 import SiteMetadata from '@theme/SiteMetadata';
+import RouteAnnouncerWrapper from './RouteAnnouncerWrapper';
+import {useLocation} from 'react-router';
 
 import './clientLifecyclesDispatcher';
 
 // TODO, quick fix for CSS insertion order
 import ErrorBoundary from '@docusaurus/ErrorBoundary';
 import Error from '@theme/Error';
 
+/* eslint-disable @typescript-eslint/ban-ts-comment */
```

Comment:
Please please please, ESLint and TS exist for a reason... You can't just disable them both...